Sources/FirebaseFirestore/Query+Swift.swift

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+@_exported
+import firebase
+@_spi(Internal)
+import FirebaseCore
+
+import CxxShim
+
+public typealias Query = firebase.firestore.Query
+
+extension Query {
+  var firestore: Firestore {
```

Comment:
Please mark this as `public` explicitly.

```suggestion
  public var firestore: Firestore {
```